app/controllers/poems_controller.rb

```diff
@@ -0,0 +1,51 @@
+class PoemsController < ApplicationController
+  before_filter :login_required
+  before_action :set_poem, only: [:edit, :update]
+  helper_method :is_mine
+
+  def index
+    @poems = Poem.where(user_id: @current_user.id).order(created_at: :desc)
+  end
+
+  def show
+    @poem = Poem.find(params[:id])
+  end
+
+  def new
+    @poem = Poem.new
+  end
+
+  def create
+    @poem = current_user.poems.build(poem_params)
+    if @poem.save
+      redirect_to @poem
+    else
+      render :new
+    end
+  end
+
+  def edit
+  end
+
+  def update
+    if @poem.update(poem_params)
+      redirect_to @poem
+    else
+      render :edit
+    end
+  end
+
+  private
+
+  def set_poem
+    @poem = current_user.poems.find(params[:id])
+  end
+
+  def poem_params
+    params.require(:poem).permit(:title, :description)
+  end
+
+  def is_mine
```

Comment:
__[rubocop]__
* Rename `is_mine` to `mine?`. [:ref](https://github.com/bbatsov/ruby-style-guide#bool-methods-qmark)